### ISSUE TITLE
Request: Condense information displayed by Activity App

### ISSUE THREAD

**Issue by sunjam** [nextcloud/activity#218]
Haven't seen an issue posted, but I'm wondering if activity could be displayed in a more easily readable way than simply listing every individual file change in Activity feed.  I really appreciate how Dropbox will start to condense "event" information once you do a certain tasks multiple times.  Condensing would optionally help activity from getting too busy when I'm dump an SD card full of photos or audio files into Nextcloud.  Perhaps clicking on the activity would display further individual details.

Example Activity feed:
John Doe added 149 new photos, 4 new documents.    
Jane Doe edited 7 documents.
Jane Doe invited 3 people to edit a file.
John Doe added 3 new users.

**Comment by nickvergessen** [comment]
We already do this a bit, if you upload multiple files they are merged into one line. However we only group until 5 entries atm so we can still display the names. Also we can't group until "149 files" because we only ever grab 50 activities at the same time, so the clients don't produce too much database load and traffic, when you open the activity page.

**Comment by sunjam** [comment]
Pardon my weak Gimp-fu.  I've condensed the Activity to combine related actions that happened within the same time frame by the same user.
![activity-example-smaller](https://user-images.githubusercontent.com/1787238/33041942-ef35b72c-cdf4-11e7-9d63-e0b79c0ca817.png)

Here is how Activity actually looks.
![screenshot_2017-11-20_11-27-37](https://user-images.githubusercontent.com/1787238/33041625-d23cd71e-cdf3-11e7-96e8-f762b8fa0f9e.png)

**Comment by Spartachetto** [comment]
I always wonder if we need two apps 

- one "_log for non technical users_": that photo was really uploaded, the system works
- one "timeline": this is a summary of all the things that involve you (exactly what @sunjam is proposing) 

the second one could use the data from the first one (the actual activity app) and reorganize it. In this way the activity app would become something that the user can use to check the single "operation" of his/her Nextcloud installation; the _timeline_ app would become the "What's going on in my cloud" app.

The timeline app would be **really, really cool** but sometimes I would need the details now available in the activity app.

Just my suggestion...

**Comment by sunjam** [comment]
@Spartachetto I believe it can all be done within the regular Activity app.  Just display condensed items, which can be easily expanded to list technical details (as it does right now).  Also, add an admin checkbox for those who want to continue seeing items listed individually.

**Comment by monsieurhannes** [comment]
I agree with @sunjam that it would be really helpful to condense the information. Same issue: I have a SD-card with photos, 300 for example, that means 60 lines of useless information for me if I want to find anything that happened before that. I personally would like to see it condensed by maps, say:

John Doe added 300 new photos to map XYZ. (with a link to the map)

I also think it should be done in one app, with the possibility to expand on entry. I have no clue how the app is build up, I just think it would be a great feature to get a better overview.
